data/core/encyclopedia/calendar.cfg

```diff
@@ -11,10 +11,8 @@
 [topic]
     id=..calendar
     title= _ "Calendar"
-    text= _ "Each year in the Wesnoth calendar is composed of 12 months. These are, in order:
+    text= _ "Each year in the Wesnothian calendar, is composed of 12 months, starting around early spring and ending in late winter. These are, in order:
```

Comment:
I know this has gone through several revisions now, but there shouldn't be a comma after 'calendar'.